src/main/java/org/springframework/cloud/dataflow/module/deployer/cloudfoundry/ApplicationModuleDeployer.java

```diff
@@ -163,6 +164,23 @@ public void run() {
 		return moduleDeploymentId;
 	}
 
+	private int deduceMemory(Map<String, String> deploymentProperties) {
+		String override = deploymentProperties.get("cloudfoundry.memory");
```

Comment:
Absolutely, but I believe I have created a separate issue for that (will look it up). We also want to use units parsing, IMHO